Source/WebKit/Shared/ios/DynamicViewportSizeUpdate.h

```diff
@@ -35,4 +35,18 @@ enum class DynamicViewportUpdateMode {
     ResizingWithDocumentHidden,
 };
 
+struct DynamicViewportSizeUpdate {
+    WebCore::FloatSize viewLayoutSize;
+    WebCore::FloatSize minimumUnobscuredSize;
+    WebCore::FloatSize maximumUnobscuredSize;
+    WebCore::FloatRect exposedContentRect;
+    WebCore::FloatRect unobscuredRect;
+    WebCore::FloatRect unobscuredRectInScrollViewCoordinates;
+    WebCore::FloatBoxExtent unobscuredSafeAreaInsets;
+    double scale;
```

Comment:
Nit - I think default initializers for some of these POD types might be nice (e.g. 1 for `scale`, 0 for `deviceOrientation` and `mEDW`?)

(Perhaps the `int32_t` could be replaced with the `WebCore::IntDegrees` typedef, too).

Reply:
good point

(and yes, I of course meant to use IntDegrees here, just screwed up :)

Reply:
thank you for catching!